tools/integration_tests/write_large_files/write_large_files_test.go

```diff
@@ -16,23 +16,44 @@
 package write_large_files
 
 import (
+	"fmt"
 	"log"
 	"os"
 	"testing"
 
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/static_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
 )
 
+func compareFileFromGCSBucketAndMntDir(gcsFile string, mntDirFile string, localFilePathToDownloadGcsFile string) (err error) {
+	err = operations.DownloadGcsObject(gcsFile, localFilePathToDownloadGcsFile)
+	if err != nil {
+		err = fmt.Errorf("Error in downloading object:%w", err)
+		return
+	}
+
+	diff, err := operations.DiffFiles(mntDirFile, localFilePathToDownloadGcsFile)
+	if diff != 0 {
+		err = fmt.Errorf("Download of GCS object %s) didn't match the Mounted local file (%s): %v", localFilePathToDownloadGcsFile, mntDirFile, err)
+		return
+	}
+
+	// Remove file after testing.
+	operations.RemoveFile(localFilePathToDownloadGcsFile)
```

Comment:
Move this up right after the successful download with a defer (`defer operations.RemoveFile(localFilePathToDownloadGcsFile)`), to avoid leaving this file behind in case the diff fails.